ccp/Cargo.toml

```diff
@@ -44,6 +44,7 @@ thiserror.workspace = true
 axum = "0.7.4"
 prometheus-client = "0.22.1"
 tokio-util = "0.7.10"
```

Comment:
Should be grouped with the other provided-by-workspace dependencies IMHO.

Reply:
Indeed, this crate is used in more that one crate.